mbuild/tests/test_xyz.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import pytest
+
+import mbuild as mb
+from mbuild.tests.base_test import BaseTest
+
+
+class TestXYZ(BaseTest):
+    def test_load(self, ethane):
```

Comment:
I think we should have two unit tests here covering cases where `top` is and is not provided to `load`.